src/Inputs/Timepicker/index.tsx

```diff
@@ -13,7 +13,7 @@ export interface TimepickerProps extends LabelLayoutProps {
     onChange?: Function;
     disabled?: boolean;
 }
-
+// @ts-ignore not sure why this is being highlighted as an issue
```

Comment:
Was line 25 being highlighted? I wonder if that is the issue

Reply:
Let me look at the error